config.go

```diff
@@ -184,8 +191,121 @@ func (c *Config) getDefaults() {
 	}
 }
 
-// CreatePythonVenv creates a virtualenv for Python scripts in ~/.ralph-cli dir.
-func CreatePythonVenv() error {
-	// TODO(xor-xor): Implement this.
+// Manifest represents the contents of a .toml file holding additional information,  which may be
+// helpful/required to run user's script (e.g., language, version, requirements etc.).
+type Manifest struct {
+	Path            string `toml:"-"`
+	Language        string
+	LanguageVersion int
```

Comment:
For Python, that would be either `2` or `3` (we need to store that info somewhere in order to be able to create virtualenv with proper interpreter).

And BTW, these manifests should be considered as initial version, which would probably change a lot in the future (at this moment, they are only used to help to create/handle Python's virtualenvs). More info on this will be available in the docs, which are coming soon (next week, I guess).